{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/settings.py

```diff
@@ -127,3 +128,42 @@
     },
 }
 {%- endif %}
+
+
+# Logging
+LOGGING = {
+    "version": 1,
+    "disable_existing_loggers": False,
+    "formatters": {
+        "structured": {
+            "()": JsonFormatter,
+            "format": "%(levelname)s %(asctime)s %(module)s %(process)d %(thread)d %(message)s",
+        },
+        "plaintext": {
+            "format": "{levelname} {asctime} {module} {process:d} {thread:d} {message}",
+            "style": "{",
+        },
+    },
+    "handlers": {
+        "console": {
+            "class": "logging.StreamHandler",
+            "formatter": config("LOG_FORMATTER", default="structured"),
+        },
+    },
+    "root": {
+        "handlers": ["console"],
+        "level": config("LOG_LEVEL", default="INFO"),
+    },
+    "loggers": {
+        "django": {
+            "handlers": ["console"],
+            "level": config("LOG_LEVEL", default="INFO"),
+            "propagate": False,
+        },
+        "{{ cookiecutter.project_slug }}": {
+            "handlers": ["console"],
+            "level": config("LOG_LEVEL", default="INFO"),
```

Comment:
Do not duplicate the LOG_LEVEL access. Define a setting like this:

LOG_LEVEL = config(...

Then use it in all places

```suggestion
            "level": LOG_LEVEL,
```